cgroups/cgroup_utils.c

```diff
@@ -291,12 +291,12 @@ static int open_if_safe(int dirfd, const char *nextpath)
 {
 	__do_close_prot_errno int newfd = -EBADF;
 
-	newfd = openat(dirfd, nextpath, O_RDONLY | O_NOFOLLOW | O_CLOEXEC);
+	newfd = openat(dirfd, nextpath, O_RDONLY | O_CLOEXEC);
 	if (newfd >= 0) /* Was not a symlink, all good. */
```

Comment:
Should probably remove this comment then? Does handling `ELOOP` separately below still make sense now?